aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -570,6 +570,18 @@ module std::features {
         is_enabled(DEFAULT_TO_CONCURRENT_FUNGIBLE_BALANCE)
     }
 
+    /// Whether the multisig v2 fix is enabled. Once enabled, the multisig transaction execution will explicitly
+    /// abort if the provided payload does not match the payload stored on-chain.
+    ///
+    /// Lifetime: transient
+    const MULTISIG_V2_FIX: u64 = 69;
```

Comment:
nit: based on the description, isn't it better to call the flag (and in Rust code) `ABORT_IF_MISMATCHED_MULTISIG_PAYLOAD`? Then feature reads easily like `abort_if_mismatched_multisig_payload_enabled()` and it is way clearer what the features is above without having to look up much?

Reply:
Done renaming it! Thank you!